doc/comparison_to_ros.md

```diff
@@ -1,12 +1,14 @@
 # Comparison to ROS
 
-* We want to move out of ROS approach to define microservices, where every component is a node and we create a massive data flow graph that gets hard to handle.
-* The benefit of that approach is that it is easy to extend and to compose from existing components, however from our experience robotic systems that are past prototype stage do not have that requirement so the cost of it outweighs the benefit.
-* We have a clear idea of what we want: determinism, easy inspectability and efficiency. This can be done with ROS but it is hard to do.
-* We want a standard message format that is not custom but widely used and highly compatible.
+ROS is an excellent framework designed to maximize prototyping speed, flexibility, and code reuse, making it ideal for research labs and early-stage startups. However, building scalable, robust products often demands a different set of priorities: static configuration, determinism, high inspectability, optimized performance, and tailored resource usage.
```

Comment:
Explain why it is hard to do thing right in ROS? -> Offering many ways to achieve the same thing results in inhomogeneous approaches, which ultimately makes inspection and troubleshooting quite difficult.